apps/web/src/components/templates/in-app-editor/InAppEditorBlock.tsx

```diff
@@ -28,42 +33,51 @@ export function InAppEditorBlock({
         const { ref, ...fieldRefs } = field;
 
         return (
-          <InAppWidgetPreview {...fieldRefs} readonly={readonly} enableAvatar={!!enableAvatar} index={index}>
-            <ContentContainerController
-              control={control}
-              index={index}
-              contentPlaceholder={contentPlaceholder}
-              readonly={readonly}
-            />
+          <InAppWidgetPreview
+            {...fieldRefs}
+            preview={preview}
+            readonly={readonly}
+            enableAvatar={!!enableAvatar}
+            index={index}
+          >
+            <>
```

Comment:
Reading the conditions of the `When` components I think the React fragment component (<>) is not needed.

Reply:
the `InAppWidgetPreview` component ask for only one element